buildSrc/src/main/groovy/com/thoughtworks/go/build/SpdxLicense.groovy

```diff
@@ -58,7 +58,7 @@ enum SpdxLicense {
   BSD_2_CLAUSE("BSD-2-Clause", "BSD 2-clause \"Simplified\" License", "The 2-Clause BSD License"),
   BSD_2_CLAUSE_FREEBSD("BSD-2-Clause-FreeBSD", "BSD 2-clause FreeBSD License"),
   BSD_2_CLAUSE_NETBSD("BSD-2-Clause-NetBSD", "BSD 2-clause NetBSD License"),
-  BSD_3_CLAUSE("BSD-3-Clause", "BSD 3-clause \"New\" or \"Revised\" License", "BSD 3-Clause", "The BSD 3-Clause License", "The 3-Clause BSD License"),
+  BSD_3_CLAUSE("BSD-3-Clause", "BSD 3-clause \"New\" or \"Revised\" License", "BSD 3-Clause", "The BSD 3-Clause License", "The 3-Clause BSD License", "BSD License 2.0"),
```

Comment:
This is the alias used for the license in Jaxen, although it's not being corrected detected/normalized by the gradle-license-report plugin so this isn't hugely helpful. Just institutionalising the knowledge here. See https://en.wikipedia.org/wiki/BSD_licenses#3-clause_license_(%22BSD_License_2.0%22,_%22Revised_BSD_License%22,_%22New_BSD_License%22,_or_%22Modified_BSD_License%22)